### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by lena-lin**

### CODE REVIEW

File: _includes/index/header.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div class="jumbotron jumbotron-fluid physikon-jumbotron px-3 text-white text-center">
   <img class="header-logo" src="img/logos-physikon/logo-pfeil-gruen.svg" alt="PhysiKon">
-  <h2 class="gray">17. - 19. April 2023 Online Vorträge <br>
+  <h2 class="gray">17. - 19. April 2023 Online-Vorträge <br>
     13 - 18 Uhr Zoom <br><br>
-    20. April 2023 Jobmesse <br> 10-16Uhr EF50
+    20. April 2023 Jobmesse <br> 10 - 16 Uhr TU Dortmund, Foyer der Emil-Figge-Straße 50
```
Comment (maxnoe) [inline _includes/index/header.html:5]: en-dash (either unicode – or `&ndash;` )

File: _includes/programm/programm.html [hunk 2]
```diff
@@ -13,11 +13,11 @@ <h3>Programm </h3>
 
 
   <p style="font-size: 1.2rem">
-    Zusätzlich zu der Jobmesse am 20.04.23 auf dem Campus der TU Dortmund bieten wir im Zeitraum 17.-19.04.23 
+    Zusätzlich zu der Jobmesse am 20.04.2023 auf dem Campus der TU Dortmund bieten wir im Zeitraum 17. - 19.04.2023
```
Comment (maxnoe) [inline _includes/programm/programm.html:16]: en-dash (either unicode – or `&ndash;` )

File: _includes/unternehmen/unternehmen.html [hunk 3]
```diff
@@ -13,24 +13,24 @@ <h2 class="h2-fontsize  py-5">Sie suchen schlaue Köpfe und motivierte Absolvent
...
+    und Online Vorträgen der teilnehmenden Unternehmen statt.
   </p> <br>
     
 
   <h5>Jobmesse</h5>
   <p style="font-size: 1.2rem">
-    Auf unserer Jobmesse haben Sie die Möglichkeit an Ihrem Messestand mit Studierenden in Kontakt zu treten.
-    Die Messe findet am Donnerstag, dem 20.04.23, 10-16Uhr auf dem Campus der TU Dortmund statt (Emil-Figge-Str. 50).
+    Auf unserer Jobmesse haben Sie die Möglichkeit, an Ihrem Messestand mit Studierenden in Kontakt zu treten.
+    Die Messe findet am Donnerstag, dem 20.04.2023, 10 - 16 Uhr auf dem Campus der TU Dortmund statt (Emil-Figge-Str. 50).
```
Comment (maxnoe) [inline _includes/unternehmen/unternehmen.html:24]: en-dash (either unicode – or `&ndash;` )

File: _includes/unternehmen/unternehmen.html [hunk 4]
```diff
@@ -13,24 +13,24 @@ <h2 class="h2-fontsize  py-5">Sie suchen schlaue Köpfe und motivierte Absolvent
...
     Die Stände können frei von Ihnen gestaltet werden. Bitte beachten Sie, das die Aufbauten innerhalb einer 
-    Fläche von 2,5x2,5m Platz finden müssen. Die Stände sind mit Strom versorgt und bei Bedarf können wir 
+    Fläche von 2,5m x 2,5m Platz finden müssen. Die Stände sind mit Strom versorgt und bei Bedarf können wir
```
Comment (maxnoe) [inline _includes/unternehmen/unternehmen.html:26]: `2.5&thinsp;m`

File: _includes/unternehmen/unternehmen.html [hunk 5]
```diff
@@ -13,24 +13,24 @@ <h2 class="h2-fontsize  py-5">Sie suchen schlaue Köpfe und motivierte Absolvent
...
-    Fläche von 2,5x2,5m Platz finden müssen. Die Stände sind mit Strom versorgt und bei Bedarf können wir 
+    Fläche von 2,5m x 2,5m Platz finden müssen. Die Stände sind mit Strom versorgt und bei Bedarf können wir 
     Stehtische zur Verfügung stellen.
   </p> <br>
 
 
   <h5>Online Vortrag</h5>
   <p style="font-size: 1.2rem">
-    Im Vorfeld der Präsenzmesse, 17.-19.04.23, bieten wir jeweils zwischen 13 und 18Uhr ein Online Programm via Zoom an. 
+    Im Vorfeld der Präsenzmesse, 17. - 19.04.2023, bieten wir jeweils zwischen 13 und 18 Uhr ein Online-Programm via Zoom an.
```
Comment (maxnoe) [inline _includes/unternehmen/unternehmen.html:33]: en-dash (either unicode – or &ndash; )

**Review by stefangri — APPROVED** [review]

**Pull request merged**